### ISSUE TITLE
Left div should be overflow:scroll

### ISSUE THREAD

**Issue by erezschatz** [thibaultduponchelle/tryperl#3]
When viewed with a wide-screen resolution, the second lesson (reversing the onion) has everything below "Start to manipulate things (onions ?! or chunky bacon :P)" hidden and the lesson is unusable. If you're going with overflow:hidden, make sure the text fits every viewscreen, or just use scroll (perhaps dynamically?)

**Comment by thibaultduponchelle** [comment]
Yes, there are still issues with css, thank you for the report. I keep the issue opened until a have a global solution.
